.github/workflows/main.yml

```diff
@@ -1,8 +1,7 @@
 name: Build and Deploy
 on:
-  push:
-    branches:
-      - main
+  release:
+    types: [created]
```

Comment:
Why this change? Then we need to create a release. Merging to `main` seems simple enough for our current use case. Am I missing somehting?

Reply:
I used it to move it temporarily.